content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,778 @@
+side of the summary displays the total points earned, and the right side displays the points that
+are available to spend.
+
+To see the various referrals and the points earned for each, click on one of the buttons directly
+beneath the avatars. The options are :guilabel:`Referrals`, :guilabel:`Ongoing`, and
+:guilabel:`Successful`.
+
+The current number of referrals who are still active in the recruitment pipeline, and have not yet
+been hired or refused, appear above the :guilabel:`Ongoing` button. The number of referrals who have
+been hired appear above the :guilabel:`Successful` button. The total number of referrals (both the
+ongoing and successful referrals combined) appear above the :guilabel:`Referrals` button.
+
+.. note::
+   To view the referrals associated with each of the categories, :guilabel:`Referrals`,
+   :guilabel:`Ongoing`, and :guilabel:`Successful`, click on the corresponding button. All the
+   referrals for that specific category are displayed.
+
+My referrals
+------------
+
+To see all the referrals (both ongoing and successful) click the :guilabel:`Referrals` button. A
+:guilabel:`My Referral` window loads. This screen displays all the referrals, with each referral in
+its own referral card.
+
+A successful referral has a green line down the left side of the card, and displays a `✓ Hired`
+image in the top right corner of the card. Referrals that are in process display an `In Progress`
+image in the top right corner.
+
+Each referral card lists the name of the applicant, the department the job position is in, the name
+of the job position, the person responsible for filling the role (recruiter), and the points earned.
+
+Points
+~~~~~~
+
+The points that can be earned for a referral are the same across all job positions. Each stage of
+the recruitment process has corresponding points assigned to it. The stages listed correspond to the
+stages configured in the *Recruitment* application.
+
+On each individual referral card, beneath the points earned is a bar chart that displays how many
+points have been earned out of the total possible points that can be earned if the applicant is
+hired.
+
+Beneath the bar chart is a list of the various recruitment stages and the points earned when the
+referral moves to that stage. If a stage has been achieved and the points have been earned, a green
+check mark appears next to the stage.
+
+.. example::
+   The pre-configured point structure for referrals is as follows:
+
+   - :guilabel:`Initial Qualification`: 1 point
+   - :guilabel:`First Interview`: 20 points
+   - :guilabel:`Second Interview`: 9 points
+   - :guilabel:`Contract Proposal`: 5 points
+   - :guilabel:`Contract Signed`: 50 points
+
+   The total points earned for a referral that is hired is 85 points. Modifications of the points
+   earned for each stage is done in the Recruitment application. Refer to the :doc:`../recruitment`
```

Comment:
Hi @larm-odoo, I am commenting on this line (552) regrading the `../recruitment` build error, can you please fix this before I review?

Thank you!

Reply:
All fixed @samueljlieber! The link currently sends you to a doc that is being updated on #5994. So when you click it, you see this page: https://www.odoo.com/documentation/17.0/applications/hr/recruitment.html
Currently, this is mostly blank, and just has a link to the Job Positions doc. But PR  #5994 is the detailed Hiring Flow doc, which is what this will be pointing to, ultimately.